roles/cifmw_cephadm/tasks/pools.yml

```diff
@@ -47,19 +47,21 @@
 
 - name: Configure the RBD trash purge scheduler
   when:
-    - cifmw_enabled_services | default([]) | intersect(['cinder_volume'])
     - cifmw_cephadm_enable_trash_scheduler | default(false)
+    - cifmw_cephadm_pools is defined
+    - cifmw_cephadm_pools | length > 0
```

Comment:
do we need a ` | default([])` here ?

Reply:
I think these checks make `| default([])` unnecessary. I think we specifically want to avoid this block if there are no `cifmw_cephadm_pools` so I think the update of the `when` is fine. I assume you were just proposing some alternative method for the `when` and not pointing out a logic problem. If there's a logic problem, then please elaborate.

Reply:
+1